### PR TITLE
chore: remove market data from trade execution

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

Removes market data from trade execution (currently used by CoW & THORSwap)

Unblocks slippage progress.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

Closes https://github.com/shapeshift/web/issues/5099

## Risk

Medium, touches receive amounts.

## Testing

Ensure the receive amount for CoW looks correct.

### Engineering

☝️

### Operations

☝️

## Screenshots (if applicable)

N/A

### CODE REVIEW

**Comment by 0xApotheosis** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5117** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5117" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5117?utm_source=stack-comment).

File: src/lib/swapper/swappers/CowSwapper/utils/helpers/helpers.ts [hunk 1]
```diff
@@ -227,6 +201,17 @@ export const getValuesFromQuoteResponse = ({
   )
 
   const rate = bnOrZero(buyAmountAfterFeesCryptoPrecision).div(sellAmountCryptoPrecision).toString()
```
Comment (0xApotheosis) [inline src/lib/swapper/swappers/CowSwapper/utils/helpers/helpers.ts:203]: No need to use the native price endpoint (a la https://github.com/shapeshift/web/issues/5099), we have enough information already.

Comment (woodenfurniture) [reply src/lib/swapper/swappers/CowSwapper/utils/helpers/helpers.ts:203]: hell yes

**Review by woodenfurniture — APPROVED** [review]
Cow quotes working (comparison taking into account market swing)
image
image

thor quotes still working
image

**Pull request merged**